### PR TITLE
Initial server

### PR DESCRIPTION
**Pull request by jankosk**
jtn pohjaa

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,31 +1,37 @@
...
+  node1:
+    build: ./server
     networks:
       - default
+    ports:
+      - 5001:5000
     environment:
-      HOSTNAME: "ping2"
-      PEERS: "ping2,ping3"
+      HOSTNAME: "node1"
```
Comment (sivuosa) [inline docker-compose.yml]: `HOSTNAME` and `TIMEOUT` are ping container specific, so they can be removed

File: server/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+WORKDIR /app
+
+ENV FLASK_APP server.py
+
+ENV FLASK_RUN_HOST 0.0.0.0
+
+COPY requirements.txt requirements.txt
+
+RUN apk update && apk add bash
```
Comment (sivuosa) [inline server/Dockerfile]: If possible, run this before any `COPY` statement to utilize Docker build cache and speed up subsequent builds. Not sure if it'll make much of a difference in our case though :P

File: server/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+RUN pip install -r requirements.txt
+
+COPY . .
+
+CMD ["flask", "run"]
```
Comment (sivuosa) [inline server/Dockerfile]: New line at EOL

File: docker-compose.yml (hunk 1 above)
Comment (jankosk) [reply docker-compose.yml]: Done.

File: server/Dockerfile (hunk 2 above)
Comment (jankosk) [reply server/Dockerfile]: Done.

File: server/Dockerfile (hunk 3 above)
Comment (jankosk) [reply server/Dockerfile]: Done.

**Pull request merged**